sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3137,24 +3137,73 @@ SIREPO.app.directive('sbatchLoginModal', function() {
 
 });
 
-SIREPO.app.directive('sbatchCoresAndHours', function(appState) {
+SIREPO.app.directive('sbatchOptions', function(appState) {
     return {
         restrict: 'A',
         scope: {
-            simState: '=sbatchCoresAndHours',
+            simState: '=sbatchOptions',
         },
         template: [
             '<div class="clearfix"></div>',
             '<div style="margin-top: 10px" data-ng-show="showCoresAndHours()">',
                 '<div data-model-field="\'sbatchHours\'" data-model-name="simState.model" data-label-size="3" data-field-size="3"></div>',
                 '<div data-model-field="\'sbatchCores\'" data-model-name="simState.model" data-label-size="3" data-field-size="3"></div>',
+                '<div data-model-field="\'sbatchQueue\'" data-model-name="simState.model" data-label-size="3" data-field-size="3"  data-ng-click="sbatchQueueFieldIsDirty = true"></div>',
             '</div>',
         ].join(''),
-        controller: function($scope) {
+        controller: function($scope, $element) {
+            $scope.sbatchQueueFieldIsDirty = false;
+            var model = appState.models[$scope.simState.model];
+            var coresPerNode = 32;
+            var max = {
+                Hours: {
+                    debug: 0.4,
+                    regular: 47
```

Comment:
Is it 47 or 48?

Reply:
48.

https://github.com/radiasoft/sirepo/pull/2511/commits/255438815245fa038582a6c8722f3ec1f8ee52bf